aes-gcm/src/lib.rs

```diff
@@ -183,23 +183,32 @@ where
     B: BlockCipher<BlockSize = U16>,
     B::ParBlocks: ArrayLength<GenericArray<u8, B::BlockSize>>,
 {
-    type NonceSize = U12;
     type TagSize = U16;
     type CiphertextOverhead = U0;
 
     fn encrypt_in_place_detached(
         &self,
-        nonce: &GenericArray<u8, Self::NonceSize>,
+        nonce: &[u8],
```

Comment:
This would require a change in the `Aead` trait. See https://github.com/RustCrypto/traits/issues/65

Reply:
Yeah. I [did that](https://github.com/RustCrypto/traits/pull/85) too

Reply:
Aah, apologies!